services/core-api/app/api/mines/explosives_permit/models/explosives_permit.py

```diff
@@ -256,6 +251,7 @@ def process_magazines(magazines, updated_magazines, type):
                 def create_permit_enclosed_letter():
                     mine = self.mine
                     # TODO: Implement a method in the document type to automatically get all read-only context values.
+                    # TODO: letter_date and letter_body are undefined, consider fixing this bug- letter_date results in NoneType has no len() error
```

Comment:
For the future.
Disabled editing for now, but this letter_date/letter_body is not sent by FE.